.github/workflows/run-tests.yml

```diff
@@ -23,9 +29,9 @@ jobs:
       - name: Run doc tests
         run: |
           cargo test --doc
-      - name: Cancel workflow runs for commit on error
-        if: failure()
-        uses: ./.github/.release/actions/actions/utils/fast-jobs-failure
+#      - name: Cancel workflow runs for commit on error
+#        if: failure()
+#        uses: ./.github/.release/actions/actions/utils/fast-jobs-failure
```

Comment:
why is this commented now? ;o

Reply:
@Xavrax right now, we didn't clone our GitHub actions to run in job. Do we really need it right now?
This action cancels other tasks and maybe at the moment of active development we would like to see all issues even if one of the checks failed.